region/maridia/inner-pink/Botwoon Energy Tank Room.json

```diff
@@ -940,6 +953,28 @@
         }
       }
     },
+    {
+      "link": [4, 4],
+      "name": "Stutter Water Shinecharge, Leave Shinesparking",
+      "entranceCondition": {
+        "comeInRunning": {
+          "speedBooster": true,
+          "minTiles": 2
+        }
+      },
+      "requires": [
+        "canStutterWaterShineCharge",
+        "canShinechargeMovementComplex",
+        {"shinespark": {"frames": 14}}
```

Comment:
In aqueduct I was able to reach the door and spark on the other side.

`canShinechargeMovementComplex` should be assumed available with stutterwatershinecharges, But my method was to do a full jump towards the door then spark.  Aqueduct has a low ceiling so an extra jump makes sense.
A `canShinechargeMovementTricky` would min/max the stutter to end closer to the door, store the spark exactly when possible, and the water hops.  Some of that is tedious to actually need to do well.  

But mostly, needing to spark back out the door seems very situational and I don't think we need to optimize it.  And on the other hand, 14shinespark frames pre transition does not leave enough energy to shinespark through LN pillar room right to left (Varia, no Etanks).  But 1 more energy would make it.  So it could enable more tight shinesparks.